src/components/backend-ai-storage-list.ts

```diff
@@ -1619,8 +1629,16 @@ export default class BackendAiStorageList extends BackendAIPage {
     });
   }
 
-  _checkProcessingStatus(status) {
-    return ['performing', 'cloning', 'deleting', 'mounted'].includes(status);
+  _isUncontrollableStatus(status: VFolderOperationStatus) {
+    return [
+      'performing',
+      'cloning',
+      'mounted',
```

Comment:
This question is not about the changes in this PR. Is it correct that the `mounted` status is an uncontrollable status and disables the share feature?
@rapsealk  @agatha197

Reply:
In accordance with the Backend.AI manager implementation, the status `VFolderOperationStatus.MOUNTED` is interpreted as both `READABLE` and `UPDATABLE`, signifying that a mounted status is still eligible for sharing. The relevant code snippet can be found at:
https://github.com/lablup/backend.ai/blob/32a0553839b489b83daa226a90ad482cda87c788/src/ai/backend/manager/api/vfolder.py#L147-L160